File: 2° Semestre/Lista Java/quadrante.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) 
  {
    Scanner ler= new Scanner(System.in);
    System.out.println("Hello world!");
    float x, y;
    System.out.printf("qual a coordenada do ponto X?");
    x= ler.nextFloat();
    System.out.printf("qual a coordenada do ponto Y?");
    y= ler.nextFloat();

    if ((y>0)&&(x<0)) System.out.printf ("está no segundo quadrante.");
    else if ((y>0)&&(x>0)) System.out.printf ("está no primeiro quadrante.");
        else if ((y<0)&&(x<0)) System.out.printf ("está no terceiro quadrante.");
              else if ((y==0)&&(x==0)) System.out.printf ("está na ORIGEM.");
                  else if ((y==0)&&(x!=0)) System.out.printf ("está no EIXO X.");
                        else if ((y!=0)&&(x==0)) System.out.printf ("está no EIXO Y.");
  }
}

File: 2° Semestre/Lista C/quadrante.c
#include <stdio.h>

int main(void) 
{
  float x, y;
  printf("qual a coordenada do ponto X?");
  scanf("%f",&x);
  printf("qual a coordenada do ponto Y?");
  scanf("%f",&y);

  if ((y>0)&&(x<0)) printf ("está no segundo quadrante.");
  else if ((y>0)&&(x>0)) printf ("está no primeiro quadrante.");
       else if ((y<0)&&(x<0)) printf ("está no terceiro quadrante.");
            else if ((y==0)&&(x==0)) printf ("está na ORIGEM.");
                 else if ((y==0)&&(x!=0)) printf ("está no EIXO X.");
                      else if ((y!=0)&&(x==0)) printf ("está no EIXO Y.");
  return 0;
}